src/inverseRobot.js

```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const values = Object.values(robot);
+  const uniqueValues = [];
+  const inversedRobot = {};
+
+  for (const value of values) {
```

Comment:
Even more - the checklist says
> [VARIABLES] - use variables for the main values so that you'll be able to reuse them and give them descriptive names.